libcloud/httplib_ssl.py

```diff
@@ -165,7 +179,8 @@ def request(self, method, url, body=None, headers=None, raw=False):
             data=body,
             headers=headers,
             allow_redirects=1,
-            stream=raw
+            stream=raw,
+            verify=self.ca_cert if self.ca_cert is not None else self.verify
```

Comment:
verify expects a boolean, but self.ca_cert is a list, so this should be `bool(self.ca_cert) ...`.